financialaid/constants.py

```diff
@@ -222,3 +222,19 @@
     "UG": 0,
     "ZW": 0
 }
+
+FINANCIAL_AID_APPROVAL_MESSAGE_BODY = """Your application for financial aid has been approved!
+Please log on to your account for more details.
+Love,
+MIT Micromasters"""
+FINANCIAL_AID_REJECTION_MESSAGE_BODY = """Your application for financial aid has been rejected.
```

Comment:
My comment seems to have been lost but, yes, there is and I'll open another issue for updating it. 

I'd like it to actually be a template, so we can customize the response. See https://goo.gl/moqYQu